### PR TITLE
[daint-gpu] SIRIUS 7.2.1

### PR DESCRIPTION
**Pull request by toxa81**
Add the latest version of SIRIUS/7.2.1

### CODE REVIEW

**Comment by lucamar** [comment]
Thanks @toxa81: you have also the script `libxc-5.1.3-CrayGNU-20.11.eb` available for `libxc 5.1.3`, in case you wish to use it as a dependency instead of `5.1.2`.

File: easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
+    ('cudatoolkit', EXTERNAL_MODULE),
+    ('intel/19.1.1.217', EXTERNAL_MODULE)
+]
+
+dependencies = [
+    ('cray-hdf5', EXTERNAL_MODULE),
+    ('GSL', '2.6'),
+    ('libxc', '5.1.2'),
```
Comment (lucamar) [inline easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb:26]:
```suggestion
    ('libxc', '5.1.3'),
```

Comment (toxa81) [reply easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb]: true! I fixed it

File: easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'CrayGNU', 'version': '20.11'}
+toolchainopts = {'opt': True, 'usempi': True, 'pic': True, 'verbose': False, 'openmp': True}
+
+source_urls = ['https://github.com/electronic-structure/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
+    ('cudatoolkit', EXTERNAL_MODULE),
+    ('intel/19.1.1.217', EXTERNAL_MODULE)
```
Comment (lucamar) [inline easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb:20]:
```suggestion
    ('intel', EXTERNAL_MODULE)
```

Comment (lucamar) [reply easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb:20]: If you are not bound to a specific Intel version, you can skip it and the default Intel compiler will be selected (in this case, the default is the version that you have selected)

File: easybuild/easyconfigs/s/SPLA/SPLA-1.3.0-CrayGNU-20.11-cuda-mkl.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True, 'verbose': False, 'opt': True}
+
+source_urls = ['https://github.com/eth-cscs/spla/archive']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('CMake', '3.14.5', '', True)]
+
+dependencies = [
+  ('cudatoolkit', EXTERNAL_MODULE),
+  ('intel/19.1.1.217', EXTERNAL_MODULE),
```
Comment (lucamar) [inline easybuild/easyconfigs/s/SPLA/SPLA-1.3.0-CrayGNU-20.11-cuda-mkl.eb:20]:
```suggestion
  ('intel', EXTERNAL_MODULE),
```

Comment (lucamar) [reply easybuild/easyconfigs/s/SPLA/SPLA-1.3.0-CrayGNU-20.11-cuda-mkl.eb:20]: If you are not bound to a specific Intel version, you can skip it and the default Intel compiler will be selected (in this case, the default is the version that you have selected)

File: easybuild/easyconfigs/s/SPLA/SPLA-1.4.0-CrayGNU-20.11-cuda-mkl.eb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True, 'verbose': False, 'opt': True}
+
+source_urls = ['https://github.com/eth-cscs/spla/archive']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('CMake', '3.14.5', '', True)]
+
+dependencies = [
+  ('cudatoolkit', EXTERNAL_MODULE),
+  ('intel/19.1.1.217', EXTERNAL_MODULE),
```
Comment (lucamar) [inline easybuild/easyconfigs/s/SPLA/SPLA-1.4.0-CrayGNU-20.11-cuda-mkl.eb:20]:
```suggestion
  ('intel', EXTERNAL_MODULE),
```

Comment (lucamar) [reply easybuild/easyconfigs/s/SPLA/SPLA-1.4.0-CrayGNU-20.11-cuda-mkl.eb:20]: If you are not bound to a specific Intel version, you can skip it and the default Intel compiler will be selected (in this case, the default is the version that you have selected)

**Review by lucamar — COMMENTED** [review]
Thanks for adding the latest `libxc` @toxa81: please see my comment about the version of the Intel compiler.

**Comment by toxa81** [comment]
@lucamar We are not bounded by specific intel version -> I fixed it as you suggested

**Review by rsarm — APPROVED** [review]

**Pull request merged**